cmd/root.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2022 The Kubernetes Authors.
+Copyright 2023 The Kubernetes Authors.
```

Comment:
no need to convert old files, only \*\***new**\*\* files you created need the 2023 since it is the current year.

Reply:
Got it, should I leave it like that or revert back to the original year?